list2md.go

```diff
@@ -99,7 +104,20 @@ func saveRanking(result []Repo) {
 	}
 	readme.WriteString(head)
 	for _, repo := range result {
+		if isDeprecated(repo.URL) {
+			repo.Description = WARNING + repo.Description
+		}
 		readme.WriteString(fmt.Sprintf("| [%s](%s) | %d | %d | %d | %s |\n", repo.Name, repo.URL, repo.Stars, repo.Forks, repo.Issues, repo.Description))
 	}
 	readme.WriteString(fmt.Sprintf(tail, time.Now().Format("2006-01-02 15:04:05")))
 }
+
+func isDeprecated(repoURL string) bool {
```

Comment:
This could be better

```
for _, repo := range deprecatedRepos {
    if (repo == repoURL) {
        return true
    }
}
```

So, you don't need to use `size`

Reply:
good point

Reply:
However, I still not understand why we do use map in this case.
O(1) vs O(n)
Memory just adds few allocation.

Reply:
Because it is for readability. For filtering the deprecated frameworks, we can just check if the URL of a specific repository is in the deprecated list. It is very intuitive.

And the performance is not important in this Go script. It would be ran just one time per day